### ISSUE TITLE
Sequences with only one annotation are not rendered correctly

### ISSUE THREAD

**Issue by haasleah** [Lattice-Automation/seqviz#60]
**Describe the bug**
It looks like there is an issue rendering sequences with only one annotation. Sometimes the end of the annotation does not line up with the end of the sequence. Additionally, the direction is not represented and you can no longer click on it to select the whole sequence (though perhaps that is intended)

**To Reproduce**
The following props were passed into the component:
```javascript
{
  "name": "BCDRBS_alt1_BD14",
  "seq": "gcgaaaaatcaataaggaggcaacaagatgtgcgaaaaacatcttaatcatgcggtggagggtttctaatg",
  "annotations": [
    {
      "start": 0,
      "end": 71,
      "direction": "FORWARD",
      "name": "RBS",
      "type": "RBS",
      "color": "#80D849"
    }
  ],
  "annotate": false,
  "searchQuery": {
    "query": "",
    "mismatch": 0
  },
  "translations": [],
  "viewer": "linear",
  "showComplement": true,
  "bpColors": {
    "a": "#000000",
    "t": "#000000",
    "c": "#000000",
    "g": "#000000"
  },
  "zoom": {
    "linear": 50
  },
  "accession": "",
  "backbone": "",
  "colors": [],
  "copySeq": {
    "key": "",
    "meta": false,
    "ctrl": false,
    "shift": false,
    "alt": false
  },
  "compSeq": "",
  "enzymes": [],
  "file": null,
  "onSearch": "onSearch()",
  "onSelection": "onSelection()",
  "searchNext": {
    "key": "",
    "meta": false,
    "ctrl": false,
    "shift": false,
    "alt": false
  },
  "showAnnotations": true,
  "showIndex": true,
  "showPrimers": true
}
```

**Expected behavior**
I expect the end of the sequence to line up with the end of the annotation and for the direction to be represented.

**Additional context**
<img width="730" alt="Screen Shot 2019-12-11 at 11 54 51 AM" src="https://user-images.githubusercontent.com/45974914/70642614-833a2580-1c0d-11ea-9732-ee71068b02c3.png">

Thanks!

**Comment by JJTimmons** [comment]
Thanks for props, made it easy to re-produce:

![Screen Shot 2019-12-11 at 2 48 47 PM](https://user-images.githubusercontent.com/13923102/70654972-63aef700-1c25-11ea-92dd-580c2794021d.png)

Added a fix where the width adjustment (accounting for multi-block linear viewers) happens in one spot.

![Screen Shot 2019-12-11 at 3 47 55 PM](https://user-images.githubusercontent.com/13923102/70659234-ae347180-1c2d-11ea-9ade-c5e2a295f6b8.png)

Am going to add a deprecation test then push a patch

**Comment by JJTimmons** [comment]
this should be addressed in v1.0.2. Please keep me posted if this persists
